go.sum

```diff
@@ -1,4 +1,5 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.31.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
do we need to commit these changes?

Reply:
I ma not sure, I noticed that this happened on @metacertain computer as well, so did anybody change go mod and did not commit the sum? should I remove it from PR?

Reply:
yep please remove it as it is not necessary for this changeset

Reply:
Ok, I have opened a new PR for this: https://github.com/ethersphere/bee/pull/423. Or how should we handle it?